### PR TITLE
To be fair auto response

### PR DESCRIPTION
**Pull request by Windos**
leverages the !tobefair function to respond with a "to be fair" gif automatically... random choice between yes and no is used so that the response only fires ~50% of the time.

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -214,6 +214,30 @@ func toBeFairResponse(ctx context.Context) string {
...
+	ctx, span = hnydiscordgo.StartSpanOrTraceFromMessage(&me, s)
+
+	span.AddField("command", "toBeFairAutoResponse")
+
+	deciders := []string{"no", "yes"}
+	decisionResponse, randNum := chooseRandom(deciders)
+	span.AddField("toBeFairAutoResponse.randomNumber", randNum)
+
+	if decisionResponse == "yes" {
+		span.AddField("toBeFairAutoResponse.RandomDecision", "Yes, responding")
```
Comment (chrisgardnersage) [inline commands.go:231]: Can we swap this and the other to `true` and `false`? Just makes querying things on Honeycomb a bit more intuitive

Comment (ChrisLGardner) [inline commands.go]: Can we swap this and the other to `true` and `false`? Just makes querying things on Honeycomb a bit more intuitive

Comment (Windos) [reply commands.go]: As in the result coming from the random generator? I'd need to make it more generic (currently it takes and returns strings explicitly.)

Could always translate the string into bool representations once they're back?

Comment (ChrisLGardner) [reply commands.go]: I was thinking of just changing `"yes, responding"` to `true` as it'll be more searchable.

Comment (Windos) [reply commands.go]: Ooh, right, gotcha

**Pull request merged**